### PR TITLE
Refactor: update plugin banner with header image details.

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves #

## Description

This PR updates the plugin banner with a new video, title and description to reflect details for the new header image. A conditional check has been added to validate the  `givewp_welcome_banner_dismiss` options timestamp against a new start date. It should show if `givewp_welcome_banner_dismiss` is not in the database or if it has a unix timestamp that is before the start date.

The video playback has also been updated. Clicking on the video container now controls the playback for the video as opposed to just the button located on the bottom left.

## Affects

Plugins page + Welcome Banner

## Visuals

https://github.com/impress-org/givewp/assets/75056371/ef0dd6e0-90a1-43b8-8c31-efeb2c37f318

## Testing Instructions
- Visit the plugin page the banner should be displaying. 
- Verify the banner video is playable.
- Close the banner -> refresh the page. Verify that no longer is visible & remains hidden.

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [x] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/Promotions/WelcomeBanner/WelcomeBanner.php [hunk 1]
```diff
@@ -13,7 +13,11 @@ public static function isShowing(): bool
 
         $option = get_option('givewp_welcome_banner_dismiss');
 
-        return $pagenow === 'plugins.php' && !$option;
+        $newStartDate = strtotime('2024-02-26 00:00');
+
+        $showBanner = ! $option || ($option < $newStartDate);
```
Comment (JasonTheAdams) [inline src/Promotions/WelcomeBanner/WelcomeBanner.php:18]: Wait, so we're going to show the banner all over again for folks that have previously seen it?

**Review by JasonTheAdams — COMMENTED** [review]
Looks pretty good, @JoshuaHungDinh! I just had one question regarding the date code.

File: src/Promotions/WelcomeBanner/WelcomeBanner.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply src/Promotions/WelcomeBanner/WelcomeBanner.php:18]: I spoke with Angela. We are looking to reintroduce the banner each time we have a big, new feature.

Reference: https://lw.slack.com/archives/C04SLRDD9CK/p1708978256130769

**Comment by JoshuaHungDinh** [comment]
Reference: https://lw.slack.com/archives/C04SLRDD9CK/p1712938181242409?thread_ts=1712926815.393089&cid=C04SLRDD9CK

**Pull request closed without merging**